### PR TITLE
Tidyup

### LINKED ISSUE #2: Mr. Happy is no longer Mr. Happy!
He's creepy halloween mr. men guy?

screen shot 2016-09-19 at 00 58 50

### PR DESCRIPTION
**Pull request by iteles**
Tiniest changes in formatting and fixes #2

### CODE REVIEW

**Comment by nelsonic** [comment]
👀  👍

File: README.md [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
+4. **If someone asks you nicely to _stop_ doing something, _stop_.** They're coming from a good place.
 
-+ **If someone asks you nicely to _stop_ doing something, _stop_.** They're coming from a good place.
+5. **Be human.** Behave like a ~~decent~~ *awesome* human being, as if someone you respect is always around! 
 
-+ **Be human.** Behave like a ~~decent~~ *awesome* human being, as if someone you respect is always around! 
+6. **Be respectful of other people's working hours.** People may be working when you're relaxing and vice versa so keep that in mind, whether you're chatting loudly in the same physical space as them or getting impatient about the resolution of an issue!
 
-+ **Be respectful of other people's working hours.** People may be working when you're relaxing and vice versa so keep that in mind, whether you're chatting loudly in the same physical space as them or getting impatient about the resolution of an issue!
+7. **Always be learning or teaching.****
```
Comment (nelsonic) [inline README.md:19]: @iteles I don't know what the effect of `****` is here... I thought the max in markdown was _four_ ...

Comment (iteles) [reply README.md:19]: Oops, obviously didn't mean to add all those to the end!

Comment (nelsonic) [reply README.md:19]: so... another PR fixing...?

**Pull request merged**